style.css

```diff
@@ -0,0 +1,651 @@
+  color: rgb(85, 83, 83);
+  font-family: 'Lato', sans-serif;
+  font-weight: 700;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  margin-top: -10px;
+}
+
+.recipe-block {
+  display: flex;
+  background-color: rgba(0, 0, 0, 0.1);
+  backdrop-filter: blur(10px);
+  margin: 10px;
+  padding: 20px;
+}
+
+.category {
+  color: #272a31;
+  font-family: 'Lato', sans-serif;
+  font-weight: 400;
+  font-size: 15px;
+  margin-left: 9%;
+  margin-top: 3%;
+}
+
+.recipe-text {
+  margin-left: 4%;
+  margin-top: 3%;
+}
+
+button.recipe_btn {
+  padding: 20px 55px;
+  width: fit-content;
+  border: none;
+  background: #ec5242;
+  margin-left: 35%;
+  margin-top: 10px;
+  margin-bottom: 50px;
+}
+
+.recipe_btn {
+  font-size: 17px;
+  font-weight: 500;
+  margin-top: 10px;
+  transition: 200ms all ease-in-out;
+  color: #fff;
+}
+
+.recipe_btn:hover {
+  background: none;
+}
+
+.recipe_btn:active {
+  background: #727ddf;
+  box-shadow: 5px 5px blue;
+}
+
+.img {
+  width: 68px;
+  height: 25px;
+  display: flex;
+  flex-direction: column;
+  margin-top: 3%;
+}
+
+.main-page a {
+  display: none;
+}
+
+.frecipe {
+  color: rgb(85, 83, 83);
+  font-family: 'Lato', sans-serif;
+  font-size: 20px;
+  display: flex;
+  justify-content: center;
+  margin-top: 0;
+}
+
+.featured {
+  background: #fff;
+  display: grid;
+}
+
+.c-card h2 {
+  margin-top: 0;
+  margin-left: 8%;
+  margin-bottom: 20px;
+  font-family: 'Lato', sans-serif;
+  color: #272a31;
+}
+
+.c-card p {
+  margin-left: 8%;
+  font-family: 'Lato', sans-serif;
+}
+
+.short {
+  margin-top: -15px;
+  color: #ec5242;
+  font-weight: 700;
+  font-size: 13px;
+  font-style: italic;
+}
+
+.l-img {
+  margin-top: -60px;
+  margin-left: 10%;
+  padding: 0;
+}
+
+.long {
+  color: #272a31;
+  font-weight: 700;
+  font-size: 12px;
+  margin-top: 0;
+}
+
+.c-card {
+  margin: 0;
+}
+
+.block1 {
+  flex-direction: row;
+  display: flex;
+  background: #fff;
+  border-radius: 4%;
+}
+
+.featurecard {
+  display: grid;
+  padding-top: 19px;
+}
+
+.line {
+  margin-left: 47%;
+  margin-top: -39px;
+  width: 5%;
+}
+
+.p-img {
+  width: 40%;
+  height: 40%;
+}
+
+button.morebtn {
+  display: flex;
+  padding: 0.1px 120px 0 130px;
+  border-radius: 1px;
+  border: 1px solid;
+  background: #fff;
+  margin-bottom: 100px;
+}
+
+.morebtn p {
+  font-size: 17px;
+  font-weight: 500;
+  font-family: 'Lato', sans-serif;
+  transition: 200ms all ease-in-out;
+  color: #272a31;
+}
+
+.morebtn:hover {
+  background: none;
+}
+
+.morebtn:active {
+  background: #727ddf;
+  box-shadow: 5px 5px blue;
+}
+
+.arrow {
+  margin-top: 24px;
+  width: 15%;
+  margin-left: 5px;
+}
+
+.Mypartners {
+  display: flex;
+  flex-direction: column;
+  background: #272a31;
+}
+
+.partners {
+  color: #fff;
+  font-family: 'Lato', sans-serif;
+  font-weight: 700;
+  font-size: 35px;
+  margin-top: -2px;
+  display: flex;
+  justify-content: center;
+}
+
```

Comment:
Please, ensure that the partners section follow the design as well. The logos are aligned on a straight line and the title has an orange mini underline. Also, kindly adjust the background color to match.

Your design:
![image](https://user-images.githubusercontent.com/20554583/190858277-8d745c1d-c5fc-4993-9a4e-7bb0f3650469.png)

Design guide:
![image](https://user-images.githubusercontent.com/20554583/190858344-79fcd55f-90e0-4aab-8dc4-182e0e6ae6a2.png)